src/huggingface_hub/constants.py

```diff
@@ -130,6 +130,11 @@ def _as_int(value: Optional[str]) -> Optional[int]:
     _as_int(os.environ.get("HF_HUB_LOCAL_DIR_AUTO_SYMLINK_THRESHOLD")) or 5 * 1024 * 1024
 )
 
+# Lock files are used to ensure the downloading in parallel won't overwrite. By default, the lock file will be deleted
+# automatically. However, the deletion would cause concurrency issues in rare cases. Set this to TRUE to prevent this
+# kind of concurrency issue.
```

Comment:
```suggestion
# Lock files are used to avoid concurrency issues when downloading files in parallel, especially to ensure a single file is not downloaded multiple times.
# By default, the lock file will be deleted automatically to keep the cache system clean. However, this deletion can cause concurrency issues in rare cases.
# Set `HF_HUB_KEEP_LOCK_FILES=1` to prevent this kind of concurrency issue.
```